pkgs/tools/admin/triton/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, installShellFiles
+, testers
+, triton
+}:
+
+buildNpmPackage rec {
+  pname = "triton";
+  version = "7.15.4";
+
+  src = fetchFromGitHub {
+    owner = "TritonDataCenter";
+    repo = "node-triton";
+    rev = "${version}";
+    hash = "sha256-RjYJT8Iw9JZzvd2d9zh2CS27qUx12nDi12k+YuTh7tk=";
+  };
+
+  npmDepsHash = "sha256-2ZTTgJ4LzmlfFoNNNPrrmna5pbREshdw5x9w5N7nasc=";
+
+  dontNpmBuild = true;
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd triton --bash <($out/bin/triton completion)
+    # Strip timestamp from generated bash completion
+    sed -i '/Bash completion generated.*/d' $out/share/bash-completion/completions/triton.bash
+  '';
+
+  passthru = {
+    tests.version = testers.testVersion {
+      package = triton;
+    };
+  };
+
+  meta = with lib; {
+    description = "TritonDataCenter CLI and client (https://www.tritondatacenter.com/triton/compute)";
+    homepage = "https://github.com/TritonDataCenter/node-triton";
+    license = licenses.mpl20;
+    platforms = platforms.all;
```

Comment:
```suggestion
```

buildNpmPackage sets this to nodejs.meta.platforms

Reply:
Fixed, thanks.